include/af/defines.h

```diff
@@ -357,6 +357,16 @@ typedef enum {
 } af_image_format;
 #endif
 
+#if AF_API_VERSION >=34
+typedef enum {
+    AF_MOMENT_M00 = 1,
+    AF_MOMENT_M01 = 2,
+    AF_MOMENT_M10 = 4,
+    AF_MOMENT_M11 = 8,
+    AF_MOMENT_FIRST_ORDER = 0xF
+} af_moment_type;
```

Comment:
Can you make this `AF_MOMENT_FIRST_ORDER = AF_MOMENT_M00 | AF_MOMENT_M01 | AF_MOMENT_M10 | AF_MOMENT_M11` ?